### PR TITLE
RATEST 178: Create A feature file for Admit Patient.Feature

### PR DESCRIPTION
**Pull request by gracebish**
Changes made:
create admitToInPatient feature file
create admitToInpatientsteps

Issue worked on:
https://issues.openmrs.org/browse/RATEST-178

### CODE REVIEW

File: src/test/resources/features/refapp-2.x/03-trackPatient/admitToInpatient.feature [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Feature: Admit To Inpatient
+
+Background:
+    	Given a user clicks on find patient record from home page
+    	
+    @selenium
+    @admitToInpatient
+    Scenario: Select Existing Patient
```
Comment (HerbertYiga) [inline src/test/resources/features/refapp-2.x/03-trackPatient/admitToInpatient.feature:8]: thanks @gracebish  ,@sherif @kdaud   how does the entire pr look to you

**Comment by kdaud** [comment]
The user story that is being captured in this PR is already covered in the [RATEST-180](https://issues.openmrs.org/browse/RATEST-180) and running in master branch.

**Comment by gracebish** [comment]
> The one user story that is being captured in this PR is already part of [RATEST-180](https://issues.openmrs.org/browse/RATEST-180) and running in master branch.

@kdaud, i remember being advised to carry on with this work during the last QA call when i paused the same question. Could you kindly advise?

**Comment by kdaud** [comment]
We had a discussion with @sherrif10 and realized we had duplicate tickets on our dashboard and were testing functionality that were already covered by selenium legacy test( I had made earlier a comment [here](https://github.com/openmrs/openmrs-contrib-qaframework/pull/75#issuecomment-902108702) that was showing duplication of work). We then consented to close the tickets that was in context. [RATEST-174](https://issues.openmrs.org/browse/RATEST-174) and [RATEST-178](https://issues.openmrs.org/browse/RATEST-174).

@gracebish the reason why the user story in this PR was captured in [RATEST-180](https://issues.openmrs.org/browse/RATEST-180) was that Transfer encounter can not be possible if a patient has not been admitted. Otherwise that workflow that is already running in master covers Admission encounter, Transfer encounter and Discharge encounter.

**Comment by kdaud** [comment]
The ticket requirement is covered in [RATEST-180](https://issues.openmrs.org/browse/RATEST-180)

**Pull request closed without merging**